src/components/reservation/ReservationModal.tsx

```diff
@@ -31,7 +31,7 @@ const ReservationModal = ({
         "reservationModalScreenReaderModalDescriptionText"
       )}
       closeModalAriaLabelText={t("reservationModalCloseModalAriaLabelText")}
-      dataCy={dataCy || "reservation-modal"}
+      dataCy={dataCy || "-reservation-modal-"}
```

Comment:
Why is this necessary? It does not look right to me.

Reply:
No, it isnt necessary, i can revert the code with the dashes :)